Source/WebKit/GPUProcess/media/cocoa/RemoteMediaPlayerProxyCocoa.mm

```diff
@@ -58,7 +58,8 @@
     // We do not want animations here.
     [CATransaction begin];
     [CATransaction setDisableActions:YES];
-    [m_inlineLayerHostingContext->rootLayer() setFrame:CGRectMake(0, 0, size.width(), size.height())];
+    RetainPtr layer = m_inlineLayerHostingContext->rootLayer();
```

Comment:
Can we add a `protectedRootLayer()` getter to `LayerHostingContext` to avoid a new local variable?

Reply:
Done.